src/appleseed/foundation/platform/path.cpp

```diff
@@ -69,11 +71,11 @@ const char* get_executable_path()
 // Windows.
 #if defined _WIN32
 
-	    const DWORD result =
+        const DWORD result =
             GetModuleFileName(
-		        GetModuleHandle(NULL),
-		        path,
-		        sizeof(path));
+                GetModuleHandle(NULL),
+                path,
+                sizeof(path));
         assert(result);
```

Comment:
Here (and below): technically `assert()`ing raw value is OK, but 1) it might be a good idea to avoid implicit casting to boolean value (i.e. assert pointers against `NULL` or `0x0` if you agree with Bjarne that macros are evil, and integers/opaque types against plain `0`), and 2) explicit comparison already used in some other places (consistency reason).